server/sonar-db-dao/src/main/resources/org/sonar/db/qualityprofile/QualityProfileMapper.xml

```diff
@@ -232,12 +233,11 @@
     FROM rules_profiles p
     JOIN project_qprofiles pp ON pp.profile_key=p.kee
     JOIN projects project ON pp.project_uuid=project.uuid AND project.kee=#{projectKey}
-    <where>
-      p.language in
-      <foreach collection="languages" open="(" close=")" item="language" separator=",">
-        #{language}
-      </foreach>
-    </where>
+    WHERE p.language in <foreach collection="languages" open="(" close=")" item="language" separator=",">#{language, jdbcType=VARCHAR}</foreach>
+    <if test="organizationUuid != null">
+      AND project.organization_uuid = #{organizationUuid, jdbcType=VARCHAR}
+      AND p.organization_uuid = #{organizationUuid, jdbcType=VARCHAR}
```

Comment:
do not use an input twice, favor using a join which could give a hint the DB engine that the value is actually the same